modules/ROOT/pages/common-builder-customizations.adoc

```diff
@@ -99,6 +99,53 @@ By default, Experience Cloud generates the URL for your portal domain. You can c
 The new domain URL displays in the *Your chosen new My Domain URl* field.
 . Click *Save*.
 
+== Configure Feed Component
+Configure the Chatter Feed components. The Feed component displays a feed of all record or group interactions, including posts, questions, system updates, and attachments.
+
+Specify values for the following properties:
+
+|===
+|Property| Details
+|Feed Type
+|*Group or Record* shows everything that is posted to a selected group or record, for example, an account or opportunity.
+
+*Discussion* feed shows everything that is posted to *Navigational Topics*.
+
+*My Feed* shows feed items that are tagged with a topic, or a person or record that you follow. My Feed also shows all of your personal site activity. You see all shared feed items unless another person shared the item with a group that you aren't a member of. With public groups, you see the feed item that you shared despite not being a member.
```

Comment:
Wouldn't you always see items that you share?

Reply:
Is this Chatter stuff described elsewhere - like in the Salesforce docs? Can you just link to it from here?